cart-service/src/lib/dovetech-response-mapper.ts

```diff
@@ -64,27 +80,15 @@ export default (
     return invalidCouponCodeResponse;
```

Comment:
Im not sure we should do this rejection if the type is Order. We have a check to ensure the total of the order matches the basket so is it a problem if the code is no longer valid? Discuss this one at stand-up...